sources/packages/api/src/route-controllers/institution-locations/institution-location.institutions.controller.ts

```diff
@@ -112,27 +119,45 @@ export class InstitutionLocationInstitutionsController extends BaseController {
    * Get all active application of a location in an institution
    * with application_status is completed.
    * @param locationId location id.
+   * @param pagination options to execute the pagination.
+   * @param archived archive value of applications requested by user.
    * @returns Student active application list of an institution location.
    */
   @HasLocationAccess("locationId")
   @Get(":locationId/active-applications")
   async getActiveApplications(
     @Param("locationId") locationId: number,
-  ): Promise<ActiveApplicationSummaryAPIOutDTO[]> {
+    @Query() pagination: ApplicationStatusPaginationOptionsAPIInDTO,
+    @Query("archived", ParseBoolPipe)
+    archived: boolean,
```

Comment:
is this `@Param(`?

Reply:
since this API is used in both available and not available endpoints. pls ignore the 2nd comment